src/Core/src/Platform/Android/MauiApplication.cs

```diff
@@ -22,9 +19,9 @@ protected MauiApplication(IntPtr handle, JniHandleOwnership ownership) : base(ha
 
 		public override void OnCreate()
 		{
-			RegisterActivityLifecycleCallbacks(new ActivityLifecycleCallbacks());
-
 			var mauiApp = CreateMauiApp();
+			RegisterActivityLifecycleCallbacks(new ActivityLifecycleCallbacks(this, mauiApp.Services));
+			RegisterComponentCallbacks(new ApplicationComponentCallbacks(this, mauiApp.Services));
```

Comment:
2 things here:

1. The order has changed, so does this matter and will it affect users?
2. I like the concept of using the callbacks instead of overriding all the methods on the application.

Reply:
I see we assumed services was around and if not we just skipped. So probably point 1. is not super valid actually.

Reply:
I think it happened during rebase